proof_of_stake/src/parameters.rs

```diff
@@ -177,4 +178,11 @@ pub mod testing {
             }
         }
     }
+
+    /// Get an arbitrary rate - a Decimal value between 0 and 1 inclusive, with
+    /// some fixed precision
+    pub fn arb_rate() -> impl Strategy<Value = Decimal> {
+        (0..100_001_u64)
```

Comment:
```suggestion
        (0..=100_000_u64)
```
special syntax for upper bound inclusive

Reply:
Thx!